src/go/configinfo/service_info.go

```diff
@@ -515,6 +518,30 @@ func (s *ServiceInfo) processBackendRule() error {
 	return nil
 }
 
+// For methods that are not associated with any backend rules, create one
+// to associate with the local backend cluster.
+func (s *ServiceInfo) processLocalBackendOperations() error {
+	for _, operation := range s.Operations {
+		method, err := s.getOrCreateMethod(operation)
```

Comment:
only "get",  we should not be "creating" a new method here.

Reply:
Done